piicatcher/generators.py

```diff
@@ -141,11 +159,11 @@ def data_generator(
         include_table_regex_str=include_table_regex_str,
         exclude_table_regex_str=exclude_table_regex_str,
     ):
-
         try:
-            columns = _filter_text_columns(
-                catalog.get_columns_for_table(table=table, newer_than=last_run)
-            )
+            columns = catalog.get_columns_for_table(table=table, newer_than=last_run)
+            if source.source_type != "bigquery":
```

Comment:
Why can't you filter the BigQuery table columns?

Reply:
The filter function filters away data type STRING because the other data sources stores strings as VARCHAR. For BigQuery, the VARCHAR equivalent is STRING. If we filter BigQuery the same way, the data source will be empty.

Reply:
Then how about you update `_filter_text_columns` to include `string` columns in addition to (var)char and text columns?

Reply:
Will try to include `string` columns again to see if it affects the other databases, if it doesn't then I will include `string` columns in addition to (var)char and text columns!